index.html

```diff
@@ -10,10 +10,23 @@
 h3 {
 	text-transform:uppercase;
 }
-li {
-	overflow:hidden;
-	padding:20px 0;
-	border-bottom:1px solid #eee;
+ul {
+	margin:0;
+	padding:0;
+}
+ul li {
+	cursor:pointer;
+	position:relative;
+	padding:12px 8px 12px 40px;
+	background:#eee;
+	font-size:18px;
+	transition:0.2s;
+}

```

Comment:
Sample: make the list items unselectable for the cleaner UX.

use `user-select:none;`